store/appStore.ts

```diff
@@ -0,0 +1,34 @@
+import { Client as Appwrite, Account, Databases } from "appwrite";
+import create from "zustand";
+import { devtools, persist } from "zustand/middleware";
+
+import { AppState, } from "utils/types";
+
+
+const useAppStore = create<AppState>()(
```

Comment:
I feel like down the line we might want to rename this userStore or something related to the user/account. AppState make me think of a state for the whole application, which I would prefer to avoid, and create stores for just each use-case with a specified related name.

Reply:
I totally agree with this. I think each store should have its own purpose. I'll rename it to userStore